src/Database/DbController/BookController.ts

```diff
@@ -0,0 +1,67 @@
+import { Op, Model } from "sequelize";
+import { BookModel } from "../Models/Book";
+import { Book } from "../index";
+import LocalBookType from "../../Types/LocalBookType";
+import sequelize from "../config";
+import { UserModel } from "../Models/User";
+
+class BookController {
+  async addBook(book: LocalBookType) {
+    try {
+      const dbBook = await Book.create({ ...book });
```

Comment:
Because I get a type error when i just pass the book -   Argument of type 'LocalBookType' is not assignable to parameter of type 'Optional<any, string>'